content/faq/_index.md

```diff
@@ -149,3 +149,55 @@ you need and us making them available for you.
 
 In the mean time we are solving these requests one by one, so [please reach out
 to us](#how-can-i-contact-you).
+
+### A command failed in packit-service: how do I reproduce it locally?
+
+We don't have an end-to-end solution to this, yet.
+
+In the meantime, you can pull our production sandbox image
+(`docker.io/usercont/sandcastle:prod`) and run the command inside. As an
```

Comment:
the exact image is specified below in `docker pull` & `docker run`, so I don't think it needs to be duplicated also here

```suggestion
and run the command inside. As an
```